taxi/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Manufacturer, Car
+
+
+class ManufacturerTests(TestCase):
+    def test_manufact_str(self) -> None:
+        manufact = Manufacturer.objects.create(name="Tesla", country="USA")
+        self.assertEqual(str(manufact), "Tesla USA")
+
+
+class DriverTests(TestCase):
+    def setUp(self) -> None:
+        self.driver = get_user_model().objects.create(
+            username="johndoe",
+            password="secret",
+            first_name="John",
+            last_name="Doe",
+            license_number="ADF12345",
+        )
+
+    def test_license_number(self) -> None:
+        self.assertEqual(self.driver.license_number, "ADF12345")
+
+    def test_driver_str(self) -> None:
+        self.assertEqual(str(self.driver), "johndoe (John Doe)")
+
+    def test_get_absolute_url(self):
+        absolute_url = self.driver.get_absolute_url()
+        self.assertEqual(absolute_url, "/drivers/1/")
+
+    def test_create_driver_without_license_shuld_raise_exception(self):

```

Comment:
There is a typo in the method name `test_create_driver_without_license_shuld_raise_exception`. It should be `test_create_driver_without_license_should_raise_exception` to correct the spelling of 'should'.